### ISSUE TITLE
Applying via Mobile with Dropbox or Box or Google Drive or iDrive

### ISSUE THREAD

**Issue by larryhemley** [bullhorn/career-portal#67]
## Current behavior
Currently with the Career Portal, iPhone users can not apply using Mobile. It actually tells iPhone users to email a link to themselves in order to complete the online application from a desktop. Is there a plan to integrate Dropbox or Box or Google Drive or iDrive or other in order for mobile applicants to apply using their resume.
-

## Expected behavior
When a mobile iPhone user applies it should give them the option to attach/upload their resume. 
-

## Steps to replicate behavior

1. From an iPhone go to the career portal and attempt to apply to a job and you will see that there is no option to apply.

**Comment by krsween** [comment]
Hey, @larryhemley—
This behavior is explained at length [here](https://github.com/bullhorn/career-portal/issues/61).

tl;dr: iOS Safari has a security setting that makes our client-side LinkedIn integration not work and a solution to this problem requires a significant change in architecture.

I'm going to close this issue, but feel free to comment questions.

**Comment by krsween** [comment]
### Update From LinkedIn
We've received an update from the LinkedIn development team that they are aware of the iOS Safari issue and are planning to develop a new version of their Apply With LinkedIn API. We will continue to provide updates as we receive them as well as provide information regarding how that will influence our roadmap.
